source/Templates/TeamCitySetup/.template.config/template.json

```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Octopus Deploy",
+  "classifications": ["Sashimi"],
+  "description": "Creates a Sashimi TeamCity setup",
+  "name": "Sashimi TeamCity",
+  "defaultName": "MySashimi",
+  "identity": "Octopus.Sashimi.TeamCity",
+  "groupIdentity": "Octopus.TeamCity",
+  "tags": {
+    "language": "C#",
+    "type": "project"
+  },
+  "shortName": "sashimitc",
```

Comment:
Rename to `sashimi-teamcity-step`?

Reply:
Is not really a step @flin-8 
Maybe just `sashimi-teamcity`

Reply:
Fixed